### ISSUE TITLE
  Why should the shuffle of dataloader be set to true？

### ISSUE THREAD

**Issue by 773041642** [qiqihaer/RandLA-Net-pytorch#21]
The sampling point is selected according to the probability value. Will it not break the corresponding relationship between the probability value and the point if the sequence of data is disrupted?
